qemu/tests/rng_bat.py

```diff
@@ -63,13 +63,27 @@ def get_driver_id(session, cmd, pattern):
         driver_id = '^&'.join(driver_id.split('&'))
         return driver_id
 
+    def check_host_read_running(target_process):
```

Comment:
`is_host_read_running` maybe easier to understand

Reply:
Modified,thanks